recipe/include-sstream.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/Contact.cxx b/Contact.cxx
+index e16a417..5844036 100644
+--- a/Contact.cxx
++++ b/Contact.cxx
+@@ -23,6 +23,8 @@ limitations under the License.
+ #include <vtkTriangleStrip.h>
+ #include <vtkArrayIteratorTemplate.h>
+ 
++#include <sstream>
```

Comment:
@zippy84 I had to add `#include <sstream>` in a number of places via a recipe patch. This should probably go into the next release of the upstream project.

Reply:
I've already seen. Thanks for the hint.